src/Data/Text/Array.hs

```diff
@@ -130,8 +134,10 @@ unsafeIndex (ByteArray arr) i@(I# i#) =
   case indexWord8Array# arr i# of r# -> (W8# r#)
 {-# INLINE unsafeIndex #-}
 
--- sizeofMutableByteArray# is deprecated, because it is unsafe in the presence of
--- shrinkMutableByteArray# and resizeMutableByteArray#.
+-- | 'sizeofMutableByteArray#' is deprecated, because it is unsafe in the presence of
+-- 'shrinkMutableByteArray#' and 'resizeMutableByteArray#'.
+--
+-- @since 2.0
```

Comment:
I'm confused by this comment, how does it relate to `getSizeofMArray`? I feel like there's some context missing in the comment.

Reply:
Should be better now, thanks.